Tests/Common/Packets/BacktestNodePacketTests.cs

```diff
@@ -102,5 +103,16 @@ public void RoundTripWithJobDates()
             Assert.AreEqual(job.PeriodStart, job2.PeriodStart);
             Assert.AreEqual(job.PeriodFinish, job2.PeriodFinish);
         }
+        [Test]
+        public void InitialCashAmount()
+        {
+            var job = new BacktestNodePacket(1, 2, "3", null, 9m, $"{nameof(BacktestNodePacketTests)}.Pepe");
+            Assert.AreEqual(job.CashAmount.Value,9m);
+
+            job.CashAmount = 10m;
+            var serialized = JsonConvert.SerializeObject(job);
+            var job2 = JsonConvert.DeserializeObject<BacktestNodePacket>(serialized);
+            Assert.AreEqual(job.CashAmount, job2.CashAmount);
```

Comment:
I think we also need a specific unit test that asserts the initial cash amount is respected, similar to `JobDatesAreRespected`. 
Also lets please add another unit tests that asserts that other currencies set by the algorithm are also set to 0, not only the account currency, the use case pointed in the previous comment